core/runner.js

```diff
@@ -367,6 +367,8 @@ class Runner {
 
     let auditResult;
     try {
+      if (artifacts.PageLoadError) throw artifacts.PageLoadError;
```

Comment:
Not an outrageous idea, although I think repeating the same error message everywhere has benefits.

I've responded to a few bugs where users ask about the "traces gatherer did not run" error message but completely miss the more detailed top level run warning that explains the problem better. I believe users (myself included sometimes) skip over the warning section and go straight to the giant wall of red text for the failure information.

Reply:
> The message could be "An error during page load prevented this audit from running. See the error message given at the top of the report" or something.

Yeah but from a JSON consumer perspective this makes less sense.

> How long are we talking here? 10x/100x than what I just wrote above?

It vary depending on what [error details](https://github.com/GoogleChrome/lighthouse/blob/e5d1b9c2491b7c4695613070b4a2cc78e8fc5456/core/lib/lh-error.js#L45) are provided. FWIW the error will be in a tooltip most of the time. If we're really worried about blowing up the metric section though, I would prefer to elide the displayed text rather than choose a more indirect albeit shorter  error message.